packages/auth/src/access/types.ts

```diff
@@ -156,24 +144,16 @@ export function isIncomingPaymentLimit(
   limit: IncomingPaymentLimit
 ): limit is IncomingPaymentLimit {
   return (
-    typeof limit.expiresAt === 'string' &&
-    typeof limit.description === 'string' &&
-    typeof limit.externalRef === 'string' &&
-    isPaymentAmount(limit.incomingAmount)
+    typeof limit.expiresAt === 'string' && isPaymentAmount(limit.incomingAmount)
   )
 }
 
 export function isOutgoingPaymentLimit(
```

Comment:
`limits` now only exists on outgoing payments.
https://github.com/interledger/open-payments/blob/main/auth-server-open-api-spec.yaml#L872-L875
Seems like at least `isIncomingPaymentLimit` could be removed.